python_modules/libraries/dagster-postgres/dagster_postgres/event_log/event_log.py

```diff
@@ -112,18 +113,20 @@ def _init_db(self) -> None:
 
     def optimize_for_webserver(self, statement_timeout: int, pool_recycle: int) -> None:
         # When running in dagster-webserver, hold an open connection and set statement_timeout
+        kwargs = {
+            "url": self.postgres_url,
+            "isolation_level": "AUTOCOMMIT",
+            "pool_size": 1,
+            "pool_recycle": pool_recycle,
+        }
         existing_options = self._engine.url.query.get("options")
-        timeout_option = pg_statement_timeout(statement_timeout)
         if existing_options:
-            options = f"{timeout_option} {existing_options}"
-        else:
-            options = timeout_option
-        self._engine = create_engine(
-            self.postgres_url,
```

Comment:
it seems like at least on sqlalchemy 1, this needs to be a positional argument (or a 'name_or_url' kwarg)